dipy/core/tests/test_geometry.py

```diff
@@ -219,6 +219,22 @@ def test_vec2vec_rotmat():
         R = vec2vec_rotmat(a, b)
         assert_array_almost_equal(np.dot(R, a), b)
 
+    a = np.array([0.33950729, 0.92041729, 0.1938216])
```

Comment:
Are these vectors where this numerical instability happens? It would be good to build a test around them if so.

Reply:
Yes. These two vectors are the examples that their dot product is slightly more than 1 and create instability:
 a = np.array([0.33950729, 0.92041729, 0.1938216])
 b = np.array([0.40604787, 0.97518325, 0.2057731])
Sure. I will build a test for these vectors.